tests/js/spec/utils/discover/fieldRenderer.spec.jsx

```diff
@@ -256,4 +264,58 @@ describe('getFieldRenderer', function () {
     // Since there is no project column, it is not wrapped with the dropdown
     expect(wrapper.find('TeamKeyTransaction')).toHaveLength(0);
   });
+
+  describe('ops breakdown', () => {
+    it('can render operation breakdowns', async function () {
+      const renderer = getFieldRenderer(SPAN_OP_RELATIVE_BREAKDOWN_FIELD, {
+        [SPAN_OP_RELATIVE_BREAKDOWN_FIELD]: 'string',
+      });
+
+      const wrapper = mountWithTheme(
+        renderer(data, {location, organization}),
+        context.routerContext
+      );
+
+      const value = wrapper.find('RelativeOpsBreakdown');
+      expect(value).toHaveLength(1);
+
+      const expectWidthAt = (width, index) => {
+        expect(value.children().children().at(index).getDOMNode().style.width).toEqual(
```

Comment:
Nit: You can refactor this into a `getWidth` function and call it to get the width. That way you can write something like

```javascript
expect(getWidth(...)).toEqual('...');
```

once for each assertion which will have a nicer failure message. Plus you can get rid of some duplication from the test below. 😄

Reply:
👍 updated! Tests look better now!